controller/users_blackbox_test.go

```diff
@@ -51,6 +54,48 @@ func (s *TestUsersSuite) SecuredServiceAccountController(identity account.Identi
 	return svc, NewUsersController(svc, s.GormDB, s.Configuration)
 }
 
+func (s *TestUsersSuite) TestDeleteUsersBadRequest() {
+	// given
+	fxt := tf.NewTestFixture(s.T(), s.DB, tf.Users(1), tf.Identities(1))
+	secureService, secureController := s.SecuredServiceAccountController(*fxt.Identities[0])
+	// when
+	emptyUsername := ""
+	test.DeleteUsersBadRequest(s.T(), secureService.Context, secureService, secureController, emptyUsername)
+}
+
+func (s *TestUsersSuite) TestDeleteUsersOK() {
```

Comment:
You don't test what happens when you have identities that share the same username.

Reply:
fixed in c8d552870fe7bc5350952b25bd2c757f27e5208b